aws-ssm-patchbaseline/.gitignore

```diff
@@ -12,9 +12,12 @@ out.java
 out/
 .settings
 .project
-
 # auto-generated files
 target/
 
 # our logs
 rpdk.log
```

Comment:
remove those whitespaces, 
if the file doesn't change at all, please remove those unnecessary changes.

Reply:
no changes here, this file should not be in Files Changed.

Reply:
We dont have any changes and now we have reverted to the previous stable changes and removed white spaces in the file